### ISSUE TITLE
Some questions about this paper

### ISSUE THREAD

**Issue by hanjiang2001** [sLdZ0306/CCR#2]
Firstly, I would like to express my gratitude for open-sourcing your code, and it is indeed a remarkable piece of work. However, I find myself puzzled by certain aspects. In regards to the loss function during training, why is the second term in Equation 12 necessary? Given that the paper aims to mitigate the impact of the masked query on reconstruction, wouldn't it be more logical to solely optimize the first term? Why is there a need to optimize the second part as well? Furthermore, I'm curious as to why a ranking loss is employed when the objective is to eliminate the influence of reconstruction. Does this imply that the reconstruction loss for negative samples is still decreasing, albeit mitigated to some extent? I eagerly await your insights on these matters.
